Assets/AnkrSDK/Plugins/WalletConnectSharp.Unity/WalletConnect.cs

```diff
@@ -303,11 +315,11 @@ public void OpenDeepLink(AppEntry selectedWallet)
 
 		public void OpenMobileWallet()
 		{
-		#if UNITY_ANDROID
```

Comment:
don't override this codestyle, setup your env to make tabulation align with the braces